assets/scss/templates/_doppler-plus.scss

```
@@ -413,60 +413,59 @@
 
   /* pricing */
 
-  .dp-wrap-price {
+  .dp-price--wrapper {
     padding: $dp-spaces--lv7;
     text-align: center;
     overflow: hidden;
 
-    .dp-old-price {
+    .dp-price-old {
       display: flex;
       justify-content: center;
       align-items: center;
 
       color: $dp-color-silver;
 
-      .dp-money {
+      .dp-price-old-money {
         margin-right: $dp-spaces--lv2;
         font-size: 12px;
         line-height: 30px;
       }
 
-      .dp-old-amount {
+      .dp-price-old-amount {
         text-decoration: line-through;
         font-size: 24px;
         line-height: 30px;
       }
     }
 
     /* small */
-    .dp-amount-small {
+    .dp-price-small-amount {
```

```suggestion
    .dp-price-small {
```
Here `dp-price-small` would be the main component, and `dp-price-small-money` and `dp-price-small-amount` subcomponents.